test/common/network/filter_manager_impl_test.cc

```diff
@@ -76,24 +76,72 @@ TEST_F(NetworkFilterManagerTest, All) {
   read_filter->callbacks_->continueReading();
 
   read_buffer_.add("hello");
-  EXPECT_CALL(*read_filter, onData(BufferStringEqual("hello"), _))
+  read_end_stream_ = false;
+  EXPECT_CALL(*read_filter, onData(BufferStringEqual("hello"), false))
       .WillOnce(Return(FilterStatus::StopIteration));
   manager.onRead();
 
   read_buffer_.add("world");
-  EXPECT_CALL(*filter, onData(BufferStringEqual("helloworld"), _))
+  EXPECT_CALL(*filter, onData(BufferStringEqual("helloworld"), false))
       .WillOnce(Return(FilterStatus::Continue));
   read_filter->callbacks_->continueReading();
 
   write_buffer_.add("foo");
-  EXPECT_CALL(*write_filter, onWrite(BufferStringEqual("foo"), _))
+  write_end_stream_ = false;
+  EXPECT_CALL(*write_filter, onWrite(BufferStringEqual("foo"), false))
       .WillOnce(Return(FilterStatus::StopIteration));
   manager.onWrite();
 
   write_buffer_.add("bar");
-  EXPECT_CALL(*write_filter, onWrite(BufferStringEqual("foobar"), _))
+  EXPECT_CALL(*write_filter, onWrite(BufferStringEqual("foobar"), false))
       .WillOnce(Return(FilterStatus::Continue));
-  EXPECT_CALL(*filter, onWrite(BufferStringEqual("foobar"), _))
+  EXPECT_CALL(*filter, onWrite(BufferStringEqual("foobar"), false))
+      .WillOnce(Return(FilterStatus::Continue));
+  manager.onWrite();
+}
+
+TEST_F(NetworkFilterManagerTest, EndStream) {
```

Comment:
You did such a great job commenting all your other tests, mind doing this one too?